website/versioned_docs/version-0.56/navigatorios.md

```diff
@@ -201,6 +201,8 @@ NavigatorIOS uses `route` objects to identify child views, their props, and navi
 
 ### `barStyle`
 
+<!-- alex ignore rule -->
```

Comment:
Would change the below values to \`default\`, \`black\` instead (i.e. wrap them in code braces because they are code values and they will pass lint